src/main/ml-modules/root/lib/SearchCriteriaProcessor.mjs

```diff
@@ -86,11 +87,7 @@ const SearchCriteriaProcessor = class {
     sortCriteria,
     valuesOnly
   ) {

```

Comment:
@clarkepeterf, I'm working up a variation of this that keeps `_requireSearchCriteriaJson` in SearchCriteriaProcessor.  Moving it to searchLib's `processSearchCriteria` is presently safe but should we ever introduce a second call to `process`, we'd have to duplicate the check.  I've run out of time for the day but plan on testing my edits tomorrow morning.

Reply:
@clarkepeterf, I concluded `requireSearchCriteriaJson` was moved to accommodate the new nested calls of `process` but found another way: those nested calls can just pass `null` in for the scope name, thereby requiring the scope from the search criteria's `_scope` property (https://github.com/project-lux/lux-marklogic/pull/370/commits/325211b2047b08f2e30b9c1c257ba58aae1180da).  Please advise if I'm creating a problem 😜

Reply:
Looks good to me